examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -158,10 +160,18 @@ class EditAttributeListModel : public ListScreen::Model
         int n;
         if (sscanf(value.c_str(), "%d", &n) == 1)
         {
+            auto & attribute = std::get<1>(std::get<1>(std::get<1>(devices[d])[e])[c])[a];
```

Comment:
Could we figure out a way to make this more readable?

Reply:
@sagar-apple ?

Reply:
Probably in a follow up PR no, @woody-apple?

Reply:
Please file one, or fix. Ideally fix just some better variable names and splitting this out? Or event a comment... (I was cleaning out these todos in our issue tree, and... we have lots ;) )

Reply:
My assumption is that the single letter var names were chosen to keep this crazy long nested tuple access short. I'll rename and see what that does.

Reply:
Looking at it, it really think if this cleanup is needed it should be a separate PR. There's 4 nested classes all using single letter member vars.

Reply:
For sure a followup - this is used in other places.

Reply:
I put this up https://github.com/project-chip/connectedhomeip/pull/4820, it's _slightly_ easier to read.